contents/blog/what-to-ask-in-interviews.md

```diff
@@ -0,0 +1,114 @@
+---
+date: 2022-06-08
+title: Really important interview questions that more engineers should ask
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["CEO diaries", "Inside PostHog"]
+author: ["james-hawkins"]
+featuredImage: ../images/blog/ceo-diary.png
+featuredImageType: full
+---
+
+Since we started PostHog, our team has interviewed 725 people.
+
+What's one thing I've taken from this?
+
+It's normal for candidates _not_ to ask harder questions about our company, so they usually miss out on a chance to (i) de-risk our company's performance and (ii) to increase the chances they'll like working here.
+
+This just doesn't feel like it should be normal. People should be able to reduce the chances they end up in a situation like [this](https://twitter.com/carnage4life/status/1532472869334110208), or [this](https://nypost.com/2022/05/26/twitter-rescinds-job-offer-to-new-hire/).
```

Comment:
Bringing 'normality' into it could be a bit insulting and the first sentence parses oddly. We could just cut it, or change it to:

```suggestion
This isn't desirable. People should be able to reduce the chances they end up in a situation like [this](https://twitter.com/carnage4life/status/1532472869334110208), or [this](https://nypost.com/2022/05/26/twitter-rescinds-job-offer-to-new-hire/).
```